packages/gasket-plugin-dynamic-plugins/README.md

```diff
@@ -0,0 +1,94 @@
+# @gasket/plugin-dynamic-plugins
+
+Gasket plugin for dynamically adding plugins to Gasket
+
+## Installation
+
+```
+npm i @gasket/plugin-dynamic-plugins
+```
+
+Update your `gasket` file plugin configuration:
+
+```diff
+// gasket.js
+
++ import pluginDynamicPlugins from '@gasket/plugin-dynamic-plugins';
+
+ export default makeGasket({
+  plugins: [
++   pluginDynamicPlugins
+  ]
+});
+```
+
+## Usage
+
+This plugin gives you the option to dynamically add plugins to Gasket after the core Gasket instance has been created.
+
+This is can be useful when you want to add plugins to Gasket only in specific environments.
+
+For example, if you have plugins for docs creation (`@gasket/plugin-docs`, `@gasket/plugin-docusarus`) that are only needed for development purposes and do not need to be included in your production code, you can dynamically load them into Gasket only when doing development locally through this plugin. You could then add plugins designated only for development to you `devDependencies` in your `package.json` file.
+
+If you want to wait until dynamic plugins have been loaded into the Gasket instance before running app code, you can do so by checking if the `isReady` property on the Gasket instance has been resolved.
```

Comment:
`isReady` should also be documented in `@gasket/core`, perhaps as a detail of the [ready lifecycle](https://github.com/godaddy/gasket/blob/30dc358ea9b1a96b926ede52c81e87c30e5a35f7/packages/gasket-core/README.md?plain=1#L105).